.github/workflows/ci.yml

```diff
@@ -18,14 +18,13 @@ jobs:
     - if: matrix.rust == 'stable'
       run: rustup component add clippy
     - if: matrix.rust == 'stable'
-      run: cargo clippy --all-features -- -D warnings
+      run: cargo clippy --all-features --tests --examples -- -D warnings
     - if: matrix.rust == 'stable' && matrix.os == 'ubuntu-latest'
       run: cargo fmt --all -- --check
-    - run: cargo build --tests --no-default-features --verbose
-    - run: cargo test --features=ci-test --verbose
+    - run: cargo check --tests --examples --no-default-features
+    - run: cargo test --all-features
     - if: matrix.rust == 'nightly'
       run: |
         cargo +nightly update -Z minimal-versions
-        cargo check --features=ci-test --verbose --all-targets
-        cargo test --features=ci-test
-      name: Check and test with minimal-versions
+        cargo check --all-features --all-targets
```

Comment:
Why check more targets here than above?

Reply:
Fair point. I’ll change the other steps, too.